server/models/index.js

```diff
@@ -1,19 +1,35 @@
-const fs = require('fs');
-const path = require('path');
-const Sequelize = require('sequelize');
+import fs from 'fs';
+import path from 'path';
+import Sequelize from 'sequelize';
+import dotenv from 'dotenv';
 
 const basename = path.basename(__filename);
-const env = process.env.NODE_ENV || 'development';
-const config = require(`${__dirname}/../config/config.json`)[env];
+const { NODE_ENV } = process.env;
 const db = {};
 
-let sequelize;
-if (config.use_env_variable) {
-  sequelize = new Sequelize(process.env[config.use_env_variable], config);
-} else {
-  sequelize = new Sequelize(config.database, config.username, config.password, config);
+dotenv.config();
+
+let DB_URL;
+switch (NODE_ENV) {
+  case 'development':
+    DB_URL = process.env.LOCAL_DATABASE_URL;
+    break;
+
+  case 'production':
+    DB_URL = process.env.REMOTE_DATABASE_URL;
```

Comment:
But REMOTE_DATABASE_URL is not defined in the `.env` file.

Reply:
Thanks for your observation. I failed to double check, I will make changes as soon as possible.